solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -1154,6 +1212,14 @@ contract StandardPoolConverter is
         emit TokenRateUpdate(poolToken, _targetToken, _targetBalance, poolTokenSupply);
     }
 
+    function safeTransfer(IERC20 token, address to, uint256 amount) private {
```

Comment:
@barakman I am moving this as part of my PR, but please note that this function (and some functions below) should include natspec.

Reply:
Done.